README.md

```diff
@@ -3,4 +3,58 @@
 [![CircleCI](https://circleci.com/gh/1Hive/delay-app.svg?style=svg)](https://circleci.com/gh/1Hive/delay-app)
 [![Coverage Status](https://coveralls.io/repos/github/1Hive/delay-app/badge.svg?branch=master&service=github)](https://coveralls.io/github/1Hive/delay-app?branch=master&service=github)
 
-The delay app enables Aragon organizations to require a configurable delay between when an intent is sent and when it is executed.
+1Hive's delay app enables Aragon organizations to require a configurable delay between when an intent is sent and when it is executed.
+
+#### 🐲 Project stage: development
+
+The Delay app is still in development. If you are interested in contributing please see our open [issues](https://github.com/1hive/x-app/issues).
+
+#### 🚨 Security review status: pre-audit
+
+The code in this repo has not been audited.
+
+## How does it work?
+
+The Delay app keeps track of the time left until the script can be forwarded, the script to forward, and the time that the script was paused at.
+
+### Initialization
+The Delay app is initialized with the `_executionDealy` parameter. This defines the default length that a user will have to wait to execute a delayed script, provided it is not paused at some point during the delay.
+
+### Roles
+
+The Delay app should implement the following roles:
+- `SET_DELAY_ROLE`: This allows for setting a delay.
+- `DELAY_EXECUTION_ROLE`: This allows for executing a delay once it is past it's delay window.
+- `PAUSE_EXECUTION_ROLE`: This allows for pausing a delay.
+- `RESUME_EXECUTION_ROLE`: This allows for resuming a paused delay.
+- `CANCEL_EXECUTION_ROLE`: This allows for cancelling a delay.
+
+### Interface
+
+The Delay app does not have a user interface.
```

Comment:
It does have a user interface. To test it you can run `npm run start:template as usual`, try to mint some tokens and the action will be delayed. Go to the delay app ui and you will se the delay card, with the time remaining and the action to execute.